lib/ui/hooks.dart

```diff
@@ -325,10 +326,14 @@ PointerDataPacket _unpackPointerDataPacket(ByteData packet) {
       kind: PointerDeviceKind.values[packet.getInt64(kStride * offset++, _kFakeHostEndian)],
       signalKind: PointerSignalKind.values[packet.getInt64(kStride * offset++, _kFakeHostEndian)],
       device: packet.getInt64(kStride * offset++, _kFakeHostEndian),
+      pointer: packet.getInt64(kStride * offset++, _kFakeHostEndian),
       physicalX: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
       physicalY: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
+      physicalDeltaX: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
+      physicalDeltaY: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
       buttons: packet.getInt64(kStride * offset++, _kFakeHostEndian),
       obscured: packet.getInt64(kStride * offset++, _kFakeHostEndian) != 0,
+      synthesized: packet.getInt64(kStride * offset++, _kFakeHostEndian) != 0,
```

Comment:
framework need to know the if the pointer event is synthesized to be able to correctly calculate the drag velocity.